controller/src/main/java/org/jboss/as/controller/transform/TransformerRegistry.java

```diff
@@ -275,11 +288,16 @@ public static class Factory {
          * Create a new Transformer registry.
          *
          * @return the created transformer registry
+         * @deprecated Superseded by {@link #create(Stability)}.
          */
+        @Deprecated(forRemoval = true)
         public static TransformerRegistry create() {
-            return new TransformerRegistry();
+            return create(Stability.DEFAULT);
         }
```

Comment:
What happens with the current usage of this method in our code? Shouldn't we at least force the creation of a transformer registry using a specific Stability level? Otherwise, we will continue using the wrong registry and we won't be able yet to add specific transformations to the subsystems.